server/paymentProviders/stripe/bitcoin.js

```diff
@@ -84,8 +87,8 @@ export default {
             hostCurrency: balanceTransaction.currency,
             amountInHostCurrency: balanceTransaction.amount,
             hostCurrencyFxRate: order.totalAmount / balanceTransaction.amount,
-            hostFeeInHostCurrency: parseInt(balanceTransaction.amount * hostFeePercent / 100, 10),
-            platformFeeInHostCurrency: order.totalAmount*0.05,
+            hostFeeInHostCurrency,
+            platformFeeInHostCurrency: Math.round(order.totalAmount * 0.05),  /* Why not dividing by 100? Is it in a different unit? */
```

Comment:
That's a good question on why it's not divided by 100. Can you look at a couple of bitcoin transactions in the database and see if fees look reasonable?

Reply:
Good call. I'll check that and I'll also double check if there's any other place that I should be rounding it up.